jailer.go

```diff
@@ -318,6 +318,9 @@ func jail(ctx context.Context, m *Machine, cfg *Config) error {
 		WithSeccompLevel(cfg.JailerCfg.SeccompLevel).
 		WithStdout(stdout).
 		WithStderr(stderr).
+		WithStdout(os.Stdout).
```

Comment:
Shouldn't this replace the WithStdout call on line 319?

Reply:
Ah this was a rebase issue. I'll go ahead and fix that.